student.js

```diff
@@ -0,0 +1,136 @@
+storage = window.localStorage;
+window.onload = createArrays();
```

Comment:
createArrays is also invoked from the head tag. Why is that?

Reply:
resolved